build.gradle.kts

```diff
@@ -17,4 +26,64 @@ tasks {
       jvmTarget = "15"
     }
   }
+
+  test {
```

Comment:
This is quite a lot of code! I'm not suggesting we change it now but do you think it's worth applying the same pattern for future projects or retrospectively? What benefits are we getting over the default dps kotlin setup?

Reply:
I am wondering if it could go into a shared library really.   I agree it's a lot. In the event of failure you get a nice summary without having to scroll, and a clear reminder about disableed tests.

```
----
Test result: FAILURE
Test summary: 16 tests, 13 succeeded, 1 failed, 2 skipped
        Failed Tests:
                should include version info()
        Skipped Tests:
                when queue not available then check is DOWN()
                when queue available then check is UP()

16 tests completed, 1 failed, 2 skipped

> Task :test FAILED

FAILURE: Build failed with an exception.
```

If it's success, you get this which we have elsewhere but it takes less code than the Kotlin one.
```

Test result: SUCCESS
Test summary: 16 tests, 16 succeeded, 0 failed, 0 skipped
```